src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -85,67 +86,85 @@ const ResearchProfile = () => {
+                  sx={{
+                    border: '2px solid',
+                    borderColor: 'secondary.light',
+                    borderRightColor: 'primary.main',
+                    borderRightWidth: '1px',
+                    width: 'fit-content',
+                    pr: '1.5rem',
+                    ':last-child': {
+                      border: 'none',
+                    },
+                  }}>
+                  <Typography sx={{ fontWeight: 'bold' }}>{getLanguageString(role.labels)} |</Typography>
+                  <AffiliationHierarchy key={organization} unitUri={organization} />
+                </Box>
+              ))}
+            </Box>
           </Box>
-        </Box>
-      ) : (
-        <Typography>{t('my_page.no_employments')}</Typography>
-      )}
-      {orcidUri && (
-        <Box sx={lineSx}>
-          <IconButton size="small" href={orcidUri} target="_blank">
-            <img src={orcidIcon} height="20" alt="orcid" />
-          </IconButton>
-          <Box sx={textContainerSx}>
-            <Typography component={MuiLink} href={orcidUri} target="_blank" rel="noopener noreferrer">
-              {orcidUri}
-            </Typography>
+        ) : (
+          <Typography>{t('my_page.no_employments')}</Typography>
+        )}
+        {orcidUri && (
+          <Box sx={lineSx}>
+            <IconButton size="small" href={orcidUri} target="_blank">
+              <img src={orcidIcon} height="20" alt="orcid" />
+            </IconButton>
+            <Box sx={textContainerSx}>
+              <Typography component={MuiLink} href={orcidUri} target="_blank" rel="noopener noreferrer">
+                {orcidUri}
+              </Typography>
+            </Box>
           </Box>
-        </Box>
-      )}
-      {registrations && (
-        <>
-          <Typography id="registration-label" variant="h2" sx={{ mt: '2rem' }}>
-            {t('common.registrations')}
-          </Typography>
-          {isLoadingRegistrations && !registrations ? (
-            <CircularProgress aria-labelledby="registration-label" />
-          ) : registrations.size > 0 ? (
-            <>
-              <RegistrationSearchResults searchResult={registrations} />
-              <TablePagination
-                rowsPerPageOptions={ROWS_PER_PAGE_OPTIONS}
-                component="div"
-                count={registrations.size}
-                rowsPerPage={rowsPerPage}
-                page={page}
-                onPageChange={(_, newPage) => setPage(newPage)}
-                onRowsPerPageChange={(event) => {
-                  setRowsPerPage(+event.target.value);
-                  setPage(0);
-                }}
-              />
-            </>
-          ) : (
-            <Typography>{t('common.no_hits')}</Typography>
-          )}
-        </>
-      )}
-    </BackgroundDiv>
+        )}
+        {registrations && (
+          <>
+            <Typography id="registration-label" variant="h2" sx={{ mt: '2rem' }}>
```

Comment:
Har ikke noe med denne PRen å gjøre, men mangel på space her har plaget meg en stund så legger inn forslag her læll :sweat_smile: 
```suggestion
            <Typography id="registration-label" variant="h2" sx={{ mt: '2rem' }} gutterBottom>
```

Reply:
Tok det i den andre PR'en, men brukte ikke gutter, så skal fike det :)